terraform/aws/components/gpc-consumer/locals_env_variables.tf

```diff
@@ -0,0 +1,124 @@
+locals {
+  environment_variables = concat(var.gpc-consumer_environment_variables,[
+    {
+      name  = "GPC_CONSUMER_SERVER_PORT"
+      value = var.gpc-consumer_service_container_port
+    },
+    {
+      name  = "GPC_CONSUMER_ROOT_LOGGING_LEVEL"
+      value = var.gpc-consumer_root_log_level
+    },
+    {
+      name  = "GPC_CONSUMER_ROOT_LOGGING_LEVEL"
```

Comment:
name of the env variable is the same as one above it, but uses different value. Which one is the correct one?

Reply:
in https://github.com/nhsconnect/integration-adaptor-gpc-consumer/blob/main/README.md there are two logging env variables. It was a typo and now fixed.

GPC_CONSUMER_ROOT_LOGGING_LEVEL | WARN | The logging level applied to the entire application (including third-party dependencies).

GPC_CONSUMER_LOGGING_LEVEL | INFO | The logging level applied to GPC Consumer Adaptor components.